src/endpoints/Coverage/types.ts

```diff
@@ -55,7 +56,9 @@ export interface UpdateRequest {
     page?: number | null;
 }
 
-export interface CreateRequest extends UpdateRequest {}
+export interface CreateRequest extends UpdateRequest {
+    integrationRunUuid?: CoverageIntegrationRun['uuid'] | null;
```

Comment:
I would rename it:

`integrationRunId?: CoverageIntegrationRun['id'] | CoverageIntegrationRun['uuid'] | null;`